lib/hammer_cli_foreman/compute_resource/gce.rb

```diff
@@ -6,7 +6,7 @@ def name
       end
 
       def compute_attributes
-        %w[machine_type image_id network external_ip]
+        %w[associate_external_ip image_id machine_type network user_data]
```

Comment:
Thank you @ofedoren for updating attributes here.  
Here, `user_data` attribute is not required as while creating an image, user can have this attribute.